src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/utils.ts

```diff
@@ -733,16 +786,18 @@ export const hasEditQuoteErrors = (
     ? hasEditQuoteBundleErrors(result)
     : hasEditQuoteError(result)
 
-export type QuoteType =
+type EditQuoteType = keyof Pick<
```

Comment:
Yep, it definitely is more type safe. If it could be done in a less confusing way that would of course be better but maybe leave it like this for now? 😅